bluesky_queueserver/manager/start_manager.py

```diff
@@ -181,6 +184,22 @@ def start_manager():
         config['kafka']['topic'] = args.kafka_topic
         config['kafka']['bootstrap'] = args.kafka_server
 
+    if args.profile_collection_path:
+        pc_path = args.profile_collection_path
+        pc_path = os.path.abspath(os.path.expanduser(pc_path))
+        if not os.path.exists(pc_path):
+            logger.error("Profile collection directory '%s' does not exist", pc_path)
+            return 1
+        if not os.path.isdir(pc_path):
+            logger.error("Path to profile collection '%s' is not a directory", pc_path)
+            return 1
```

Comment:
I see these 2 checks, I think it's fine to keep them.

Reply:
There are two checks because there are two separate error messages.

Reply:
I see, it's the same place. I also do the same checks in the function that loads the profiles, but I think it should be checked in both places.